docs/configuration.md

```diff
@@ -169,5 +169,108 @@ php artisan up
+
+> {note} These config options are advanced config options. Do not change them unless you know what you are doing. Some of them may be unused due to internal structures.
+
+|Name|Description|Default|
+|---|---|---|
+|`LIVEWIRE_ENABLED`|Enable experimental Livewire frontend|`false`|
+|`ASSET_URL`|URL for assets|`null`|
+|`APP_CIPHER`|The app's cipher suite|`AES-256-CBC`|
+|`MEMCACHED_PERSISTENT_ID`|Persistent ID for memcached|`null`|
+|`MEMCACHED_USERNAME`|Username for memcached|`null`|
+|`MEMCACHED_PASSWORD`|Password for memcached|`null`|
+|`MEMCACHED_HOST`|Host for memcached|`127.0.0.1`|
+|`MEMCACHED_PORT`|Port for memcached|`11211`|
+|`AWS_ACCESS_KEY_ID`|Access key ID for AWS, used for dynamodb|`null`|
+|`AWS_SECRET_ACCESS_KEY`|Secret access key for AWS, used for dynamodb|`null`|
+|`AWS_DEFAULT_REGION`|Default AWS region, used for dynamodb cache driver|`null`|
+|`DYNAMODB_CACHE_TABLE`|Cache table for dynamodb cache driver|`cache`|
+|`DYNAMODB_ENDPOINT`|Endpoint for dynamodb cache driver|`null`|
+|`DATABASE_URL`|Database URL|`null`|
+|`DB_FOREIGN_KEYS`|Enable foreign key constraints|`true`|
+|`DB_SOCKET`|UNIX socket of database|empty string|
+|`MYSQL_ATTR_SSL_CA`|File path to the SSL certificate authority|`null`|
+|`REDIS_CLIENT`|Redis client|`phpredis`|
+|`REDIS_CLUSTER`|Redis cluster|`redis`|
+|`REDIS_PREFIX`|Redis prefix|the value of `APP_NAME`|
+|`REDIS_URL`|Redis URL|`null`|
+|`REDIS_DB`|Redis database|`0`|
+|`REDIS_CACHE_DB`|Redis cache database|`1`|
+|`BCRYPT_ROUNDS`|Rounds for bcrypt hashing algorithm|`10`|
+|`WEBAUTHN_NAME`|Name for Webauthn devices|the value of `APP_NAME`|
+|`WEBAUTHN_ID`|ID for Webauthn devices|`null`|
+|`WEBAUTHN_ICON`|Icon for Webauthn devices|`null`|
+|`WEBAUTHN_CACHE`|Cache for Webauthn devices|`null`|
+|`LOG_SLACK_WEBHOOK_URL`|Webhook URL for slack logging|`null`|
+|`PAPERTRAIL_URL`|URL for papertrail logging|`null`|
+|`PAPERTRAIL_PORT`|Port for papertrail logging|`null`|
+|`LOG_STDERR_FORMATTER`|Formatter for logging to stderr|`null`|
+|`MAIL_LOG_CHANNEL`|Log channel for mails|`null`|
+|`REDIS_QUEUE`|Redis queue|`default`|
+|`SQS_PREFIX`|SQS prefix|`https://sqs.us-east-1.amazonaws.com/your-account-id`|
+|`SQS_QUEUE`|SQS queue name|`your-queue-name`|
+|`SQS_SUFFIX`|SQS suffix|`null`|
+|`QUEUE_FAILED_DRIVER`|driver for failed queues|`database-uuids`|
+|`MAILGUN_DOMAIN`|Domain to mailgun|`null`|
+|`MAILGUN_SECRET`|Serect for mailgun|`null`|
+|`MAILGUN_ENDPOINT`|Endpoint to mailgun|`api.mailgun.net`|
+|`POSTMARK_TOKEN`|Token for postmark|`null`|
+|`SESSION_CONNECTION`|Session connection for `database` and `redis` drivers|`null`|
+|`SESSION_STORE`|Store for sessions, affects `apc`, `dynamodb`, `memcached` and `redis` drivers|`null`|
+|`SESSION_COOKIE`|The cookie's name|the value of `APP_NAME` + `_session`|
+|`SESSION_DOMAIN`|Session cookie domain|`null`|
+|`SESSION_SECURE_COOKIE`|Cookies only via HTTPS|`null`|
+|`VIEW_COMPILED_PATH`|Where to store compiled Blade templates|`framework/views`|
+
+
```

Comment:
I wonder whether we should change our defaults to match the example. In theory we could move all the documentation of it there, but we change it rarely enough that being in two places shouldn't be a major issue. We'll have to remember this exists though.

Reply:
I wonder if we should really list all the configuration options explicitly. While I appreciate the effort it will soon get outdated.

Reply:
Do all of these actually _do_ anything for Lychee? Say, what is `ASSET_URL` used for? I'm of the opinion that if a setting simply doesn't apply to Lychee at all, we should skip it.